templates/alma-item-list_item.tpl.php

```diff
@@ -5,10 +5,8 @@
  * Template file for single item list item.
  */
 ?>
-<div class="alma-item-list-item item-local_id-<?php print $local_id; ?>">
-  <div class="ting-cover ting-cover-processing" data-local_id="<?php print $local_id; ?>">
-    <a href="/ting/object/<?php print $faust; ?>"></a>
-  </div>
+<div class="alma-item-list-item item-local_id-<?php print $faust; ?>">
+  <?php print $image; ?>
```

Comment:
As you can see here https://github.com/easySuite/alma_item_list/pull/3/files#diff-2008b07ce66bef30a72c4aa4b479d262R154
I used renderable array, which use "ting_object_cover" theme function, which will render image.